opm/parser/eclipse/EclipseState/Schedule/MSW/SegmentSet.cpp

```diff
@@ -21,6 +21,11 @@
 #include <cmath>
 #include <map>
 
+#ifdef _WIN32
+#define _USE_MATH_DEFINES 
+#include <math.h>
```

Comment:
Here is an example of moving from M_PI to boost constants

https://github.com/ddemidov/vexcl/commit/696d74a44d406e979ebbe37dc4a1bd924e0d8358